core/ui/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/ui/UserMessage.kt

```diff
@@ -25,10 +26,18 @@ fun SnackbarMessageEffect(
 ) {
     userMessageStateHolder.messageUiState.userMessages.firstOrNull()?.let { userMessage ->
         LaunchedEffect(userMessage) {
-            val snackbarResult: SnackbarResult = snackbarHostState.showSnackbar(
-                message = userMessage.message,
-                actionLabel = userMessage.actionLabel,
-            )
+            val snackbarResult: SnackbarResult = if (userMessage.duration == null) {
+                snackbarHostState.showSnackbar(
```

Comment:
Could you elaborate on why the snackbar doesn't disappear?  I thought the default value of snackbarHostState.showSnackbar's duration is Short. So I thought it doesn't have change 👀

Reply:
The reason why the snack bar did not disappear is because the duration of snackbar is not set anywhere in the application, and default value for duration of `showSnackbar` in `Material3` is infinite if there is an `actionLabel`, and 4 seconds otherwise, and this time snackbar had an `actionLabel` set.

![image](https://github.com/DroidKaigi/conference-app-2023/assets/60963155/2a26e84f-d3b8-41f0-9164-4e96c32fa9da)

Reply:
That makes sense!